pkg/hintrunner/hinter/scope.go

```diff
@@ -84,17 +105,25 @@ func (sm *ScopeManager) GetVariableValueAsBigInt(name string) (*big.Int, error)
 		return nil, err
 	}
 
-	valueBig, ok := value.(*big.Int)
-	if !ok {
-		return nil, fmt.Errorf("value: %s is not a *big.Int", value)
-	}
+	// valueBig, ok := value.(*big.Int)
```

Comment:
It would be cool if we removed the comments, or add some comment if this is still needed in the future (draft PR)

Reply:
Done